packages/math-input/src/components/keypad/mobile-keypad.tsx

```diff
@@ -39,21 +29,28 @@ type Props = {
 
 type State = {
     containerWidth: number;
-    hasBeenActivated: boolean;
     keypadConfig?: KeypadConfiguration;
     keyHandler?: KeyHandler;
     cursor?: Cursor;
 };
 
+/**
```

Comment:
Moving this here allows this comment to be surfaced when you hover usages of this component (at least in VSCode).